changelog/unreleased/bugfix-local-logout

```diff
@@ -0,0 +1,8 @@
+Bugfix: Local logout if IdP has no logout support
+
+Some IdPs don't support a logout endpoint. In those cases the web UI ran into a fatal error an showed an empty screen without
```

Comment:
```suggestion
Some IdPs don't support a logout endpoint. In those cases the web UI ran into a fatal error and showed an empty screen without
```

Reply:
Really need to clean my keyboard 😆 this has been a frequent issue lately :trollface: